axelar-chains-config/info/devnet-amplifier.json

```diff
@@ -1494,6 +1494,8 @@
     "grpc": "devnet-amplifier.axelar.dev:9090",
     "tokenSymbol": "AMPLIFIER",
     "gasPrice": "0.00005uamplifier",
-    "gasLimit": "auto"
+    "gasLimit": "auto",
+    "depositValue": "100000000",
+    "rewardAmount": "1000000uamplifier"
```

Comment:
Is this EVM specific? Don't think we should use a static value since it depends on the chain and AXL price?

Reply:
But rewards can be configured differently for different chains, it's just that we aren't doing that rn. In fact as part of the smaller verifier set plan, we're looking at setting different rewards rates on chains now

Reply:
Also, govProposalRewardAmount isn't a great name. Deposit amount is for gov proposals. Reward amount is for chain verifiers, so maybe verifierRewardPoolPerChain